manifest/manifest.go

```diff
@@ -195,7 +195,7 @@ func MatchesDigest(manifest []byte, expectedDigest digest.Digest) (bool, error)
 }
 
 // AddDummyV2S1Signature adds an JWS signature with a temporary key (i.e. useless) to a v2s1 manifest.
-// This is useful to make the manifest acceptable to a Docker Registry (even though nothing needs or wants the JWS signature).
+// This is useful to make the manifest acceptable to a container registry (even though nothing needs or wants the JWS signature).
```

Comment:
This is (mostly() a reference to a specific implementation.

Reply:
Should we say older container registry, previous container registry?  Or is the a specific version of the Docker distribution that you want to reference?

Reply:
The commit message for this function refers to docker/distribution.

I have no idea whether other implementations accept unsigned v2s1.